controller.go

```diff
@@ -206,6 +196,66 @@ func New(cfgOptions ...config.Option) (NetworkController, error) {
 	return c, nil
 }
 
+func (c *controller) ReloadConfiguration(cfgOptions ...config.Option) error {
+	// For now we accept the configuration reload only as a mean
+	// to provide a global store configuration after boot
+	c.Lock()
+	for _, ds := range c.stores {
+		if ds.Scope() == datastore.GlobalScope {
+			c.Unlock()
+			return types.ForbiddenErrorf(
+				"cannot accept new configuration because network controller has a global scope datastore configured already")
+		}
+	}
+	// Now accept the new config
+	c.cfg = config.ParseConfigOptions(cfgOptions...)
+	c.Unlock()
+
+	if err := c.initStores(); err != nil {
```

Comment:
Yes, it is not an issue. It simply replaces the local ds client with an exact copy in the `stores` list.